src/testcafe-runner.js

```diff
@@ -26,32 +26,36 @@ async function run (runCfgPath, suiteName) {
     browserName = suite.browser;
     let testCafeBrowserName = process.env.SAUCE_VM ? browserName : supportedBrowsers[browserName.toLowerCase()];
     if (process.env.SAUCE_VM && process.env.SAUCE_BROWSER_PATH) {
-      testCafeBrowserName = `path:\`${process.env.SAUCE_BROWSER_PATH}\``;
+      testCafeBrowserName = process.env.SAUCE_BROWSER_PATH;
     }
     if (!testCafeBrowserName) {
       throw new Error(`Unsupported browser: ${testCafeBrowserName}.`);
     }
 
-    results = await runner
+    const runnerInstance = runner
       .src(path.join(projectPath, suite.src))
       .browsers(testCafeBrowserName)
       .concurrency(1)
       .reporter([
         { name: 'xunit', output: path.join(assetsPath, 'report.xml') },
         { name: 'json', output: path.join(assetsPath, 'report.json') },
         'list'
-      ])
-      .video(assetsPath, {
+      ]);
+
+    if (!process.env.SAUCE_VM || process.env.SAUCE_VIDEO_RECORD) {
```

Comment:
Hm. In cypress we use `SAUCE_CYPRESS_VIDEO_RECORDING`. Though I don't mind it being the more generic format of `SAUCE_VIDEO_RECORD` :)

Just wanted to point it out ℹ️